packages/react-link/src/components/Link/Link.test.tsx

```diff
@@ -16,7 +16,7 @@ describe('Link', () => {
   });
 
   isConformant({
-    Component: Link,
+    Component: Link as React.ComponentType,
```

Comment:
I think it'd work to specify LinkProps as the template argument instead of adding this cast.
```ts
isConformant<LinkProps>({
  Component: Link,
```

Reply:
That's true, just changed that.